holidayextras/brands
# FP-3342 - Amending progress-tracker border to white

JIRA: https://hxshortbreaks.atlassian.net/browse/FP-3342

#### What does this PR do? (please provide any background)
The JIRA was to change the styling from FrightNight back to ThorpeDark. The progress tracker borders were showing as black. However speaking with Jodie these can be changed to white as look nicer. I have amended the border colour to white for the SummaryBar variant.

#### How can this be tested?
`cd pattern-library && npm link ../brands/`

- link `@holidayextras/brand-thorpepark-dark` to `the wall` from `brands` 
- make  a change to the stylesheet in `the-wall` so the styles update
- Check that the booking flow navigation looks like in the screenshot below. It should NOT have black arrow colours but instead be white.

#### Any tech debt?
No

#### Screenshots / Screencast
Should be as below for SummaryBar variant.
<img width="1173" alt="screen shot 2017-11-09 at 13 39 06" src="https://user-images.githubusercontent.com/32061182/32608166-63464226-c553-11e7-9201-ed9e08fbbe8e.png">


#### What gif best describes how you feel about this work?
![]()


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
